pom.xml

```diff
@@ -124,12 +125,12 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
+            <version>4.5.3</version>
```

Comment:
Better to use the [apache-httpcomponents-client-4-api](https://github.com/jenkinsci/apache-httpcomponents-client-4-api-plugin) plugin instead if possible.

Reply:
@dwnusbaum agreed, but here my only goal was to get past the RequireUpperBounds failure. IMO what you're saying is definitely the way to go, but should be a dedicated ticket.

Reply:
@batmat Agreed, I think it makes sense to keep this PR minimal and handle migrating to the API plugin separately.

Reply:
Agreed